meta-mender-raspberrypi: Adjust to U-Boot 2025.04

leon-anavi opened the pull request:
Update the Raspberry Pi patches to align with U-Boot version 2025.04, following the recent upgrade to this version in layer meta-lts-mixins branch scarthgap/u-boot:
https://git.yoctoproject.org/meta-lts-mixins/commit/?h=scarthgap/u-boot&id=68856926465b56b98599ab2506fceb0d1bcfad5b

This work was sponsored by GOVCERT.LU.

Changelog: Fix patches for U-Boot 2025.04

leon-anavi committed:
meta-mender-raspberrypi: Adjust to U-Boot 2025.04

Update the Raspberry Pi patches to align with U-Boot version
2025.04, following the recent upgrade to this version in
layer meta-lts-mixins branch scarthgap/u-boot:
https://git.yoctoproject.org/meta-lts-mixins/commit/?h=scarthgap/u-boot&id=68856926465b56b98599ab2506fceb0d1bcfad5b

This work was sponsored by GOVCERT.LU.

Changelog: Fix patches for U-Boot 2025.04

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

TheYoctoJester reviewed (changes requested):
Hi @leon-anavi,

Gave it a quick build and looks good. Can you just also add revision bumps to `meta-raspberrypi` and `meta-lts-mixins` in `kas/include/raspberrypi.yml` so the PR provides a coherent state? Thanks!

leon-anavi committed:
kas/include/raspberrypi.yml: Update

Update layer meta-raspberrypi and meta-lts-mixins branch
scarthgap/u-boot with U-Boot 2025.04.

This work was sponsored by GOVCERT.LU.

Changelog: Update meta-raspberrypi and meta-lts-mixins

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>

leon-anavi commented:
@TheYoctoJester, thank you for reviewing and merging this pull request.

FIY following the update of the KAS configuration files, I successfully ran a build from scratch for Raspberry Pi 5, which completed without any errors.
